utility/generate_omap_entries.py

```diff
@@ -0,0 +1,50 @@
+"""
+Module used to:
+1. Create objects on pool,
+2. generate specified amount of dummy key value pairs, add it as attributes to the object,
+ thereby increasing the omap entries on the pool
+"""
+
+# !/usr/bin/env python
+from __future__ import print_function
+
+import os
+import sys
+
+from rados import Rados, WriteOpCtx
+
+
+def run(args):
+    if len(args) < 4:
+        print(
+            "usage: python3 generate_omap_entries.py <pool_name> <init_count> <end_count> <num_keys>"
+        )
+        sys.exit(1)
+    pool = args[1]
+    start = int(args[2])
+    end = int(args[3])
+    keys_per_object = int(args[4])
+    keys = tuple(["key_" + str(x) for x in range(keys_per_object)])
+    values = tuple(["value_" + str(x) for x in range(keys_per_object)])
+
+    with Rados(conffile="") as cluster:
+        with cluster.open_ioctx(pool) as ioctx:
+            prefix = "omap_obj_" + str(os.getpid()) + "_"
+            for i in range(start, end):
+                with WriteOpCtx(ioctx) as write_op:
+                    ioctx.set_omap(write_op, keys, values)
+                    ioctx.operate_write_op(write_op, prefix + str(i))
+                    print(
+                        "wrote",
+                        (i + 1) * keys_per_object,
+                        "of",
+                        (end - start) * keys_per_object,
+                        "omap entries",
+                        end="\r",
+                    )
+                    sys.stdout.flush()
+    print("\nDone!")
+
+
+if __name__ == "__main__":
+    run(sys.argv)
```

Comment:
Can we use `docopt`? This would allow enable us to understand the expected arguments and known how to execute the script.

Reply:
It would be good to handle `run` in a `try... catch` block and exit with the right codes.

Reply:
made changes